autoextract_spiders/sessions.py

```diff
@@ -0,0 +1,29 @@
+
+from crawlera_session import RequestSession as _Session
+
+
+def update_redirect_middleware(settings):
+    if settings.getbool('CRAWLERA_ENABLED'):
+        redirect_mware = 'scrapy.downloadermiddlewares.redirect.RedirectMiddleware'
```

Comment:
Just a convenient helper to move another piece of crawlera-only logic to the single place